package lab.user_service.security.exceptions;

public class UserNotFoundException extends RuntimeException {
}
